<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="openProperty">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Objekt öffnen...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="newProperty">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Neues Verwaltungsobjekt anlegen...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="deleteProperty">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Verwaltungsobjekt löschen...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="systemSettings">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Systemeinstellungen</string>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="exitApplication">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Programm beenden</string>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
